src/Language/PureScript/Errors.hs

```diff
@@ -84,6 +84,7 @@ errorCode em = case unwrapErrorMessage em of
   CannotGetFileInfo{} -> "CannotGetFileInfo"
   CannotReadFile{} -> "CannotReadFile"
   CannotWriteFile{} -> "CannotWriteFile"
+  TypeConstructorAlias{} -> "TypeConstructorAlias"
```

Comment:
After this is merged, can you please add docs to the docs repo for this error too?